ha/core/controllers/pcs/service_controller.py

```diff
@@ -82,3 +99,24 @@ def status(self, service: str, nodeids: list = None) -> dict:
                 status: Succeeded, Failed, InProgress
         """
         raise HAUnimplemented("This operation is not implemented.")
+
+    def clear_resources(self, node_id: str):
```

Comment:
is it different than start ?

can we rename to start

Reply:
node_id or nodeid

Reply:
Here we just removing contraints. i think start() contains enabling of resources.
node_id : Private fqdn define in conf store.